### DISCUSSION TITLE
Question on the lines when detecting and recognizing text.

### DISCUSSION THREAD

**Discussion by HuSSonO3** [mindee/doctr#1796] [Q&A] [answered]
Hayoo!,

I am trying to read a document where the horizontal part of it is all connected.
for exp:
![image](https://github.com/user-attachments/assets/2db46b41-f85a-44d1-8092-7782547ad335)

i want the model to assume that the total with the number is the whole line
but the lines that get recognized are very close words only not far apart ones, is there a way to do that?
Maybe just look for every box on the same x level and group it as a line.

**Comment by felixdittrich92** [comment, marked as answer]
Hi @HuSSonO3 :wave:,

You can pass a higher value for `paragraph_break` to the `ocr_predictor` the default is `0.035` so if you pass for example `1` instead it should merge every horizontal in one line.

Example:
```python
model = ocr_predictor(pretrained=True, paragraph_break=1)
```

Hope this helps :)

Best,
Felix

**Reply by HuSSonO3** [reply]
Thank you so much!.
